README.md

```diff
@@ -85,18 +85,19 @@ possible to debug the connector.
 
 ## Developing your Connector
 
-We recommend that you just modify the template connector directly. This shields you from having to recreate all of the
+We recommend that you just modify the template connector directly. This shields you from having to recreate all the
 configuration that is required to correctly build and debug a new project. Visual Studio has excellent
 renaming/refactoring features that make it easy. For example, you can click on any item in VS's *Solution Explorer*
 and press *F2* to change it to something appropriate for your company and this particular connector.
 
 Once you are ready to start developing, just open the `MyConnector.cs` and `MyConnection.cs` files in VS and start
 reading through the heavily-annotated source code. The template connector uses a small class called
-`DatasourceSimulator`. You'll know you've removed all of the template-specific code when you can delete this file from
+`DatasourceSimulator`. You'll know you've removed all the template-specific code when you can delete this file from
 the project and still build without errors.
 
 Any log messages you create using the `Log` property on `ConnectorServiceV2` and `DatasourceConnectionServiceV2` will go
-to the console window and to the `csharp/Seeq.Link.SDK.Debugging.Agent/bin/Debug/log/net-debugging-agent.log` file.
+to the console window and to the `csharp/Seeq.Link.SDK.Debugging.Agent/bin/x64/Debug/log/net-debugging-agent.log` file 
+(replace `x64` with `x86` if you're on a 32-bit machine).
```

Comment:
Maybe we shouldn't ecourage the third party developers to build connectors for 32-bit platform.
I think we currently have one single connector that still need x86 and hopefully we will discontinue this in the future